### DISCUSSION TITLE
Manual Installation Failed on arm server

### DISCUSSION THREAD

**Discussion by SeldonHarry** [nextcloud/all-in-one#3747] [Questions] [answered]
I have followed the instructions in manual installation and use docker-compose v2.22 to compose the yaml file. After successfully pulling and creating, it failed to start in creating container and running. It threw errors "manual-install-nextcloud-aio-nextcloud-1    | Waiting for the database to start." on end. It seems that database can not be woke up?

Full errors are:
manual-install-nextcloud-aio-database-1        | 2023-11-15 05:34:00.997 UTC [31] FATAL:  password authentication failed for user "oc_nextcloud"
manual-install-nextcloud-aio-database-1        | 2023-11-15 05:34:00.997 UTC [31] DETAIL:  Role "oc_nextcloud" does not exist.
manual-install-nextcloud-aio-database-1        | 	Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
manual-install-nextcloud-aio-nextcloud-1       | psql: error: connection to server at "nextcloud-aio-database" (172.18.0.4), port 5432 failed: FATAL:  password authentication failed for user "oc_nextcloud"
manual-install-nextcloud-aio-nextcloud-1       | Waiting for the database to start...
manual-install-nextcloud-aio-notify-push-1     | Waiting for Nextcloud to start...
manual-install-nextcloud-aio-apache-1          | Waiting for Nextcloud to start...

**Comment by szaimen** [comment, marked as answer]
Hi, this looks like a hickup. Please reset things properly and try again.
